components/comparision/ComparisionTable.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+  <section class="container text-center">
+    <h2 class="section__heading">{{ heading }}</h2>
+    <p class="section__subtitle">
+      {{ description }}
+    </p>
+    <div class="table-responsive">
+      <table class="table text-start comparision__table mx-auto">
+        <thead class="bg__light-primary">
+          <tr>
+            <th scope="col">Features</th>
+            <th
+              scope="col"
+              v-for="header in headers"
+              :key="header"
+              class="text-center"
+            >
+              {{ header }}
+            </th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr v-for="c of comparisions" :key="c[0]">
```

Comment:
```suggestion
          <tr v-for="c of comparisions" :key="c.title">
```
reason for not using c.title as a key